java/kotlin-extractor2/src/main/kotlin/entities/Expression.kt

```diff
@@ -211,6 +215,64 @@ OLD: KE1
         }
 */
 
+private fun isFunction(
```

Comment:
The KE1 equivalent of this function has some extra parameters, such as nullability, so adding `Named` would not necessarily make sense.

Reply:
OK; I don't like the name `isFunction`, because it sounds like it tells you whether something is a function or something else (e.g. a class), but perhaps it's too early to pick the best name.